packages/backend-defaults/src/entrypoints/rootHttpRouter/http/MiddlewareFactory.test.ts

```diff
@@ -254,5 +241,57 @@ describe('MiddlewareFactory', () => {
 
       expect(childLogger.error).toHaveBeenCalled();
     });
+
+    it('should log incoming requests', async () => {
+      const app = express();
+      app.use(middleware.logging());
+      app.get('/', (_req, res) => res.send('Hello World'));
+
+      await request(app).get('/').expect(200);
+
+      expect(logger.info).toHaveBeenCalledWith(
+        expect.stringContaining(
+          '[20/Nov/2024:00:00:00 +0000] "GET / HTTP/1.1" 200 11 "-" "-"',
+        ),
+        {
+          type: 'incomingRequest',
+          date: 'Wed, 20 Nov 2024 00:00:00 GMT',
```

Comment:
i don't see the conversion being done - i meant, if we keep this at all (i argue that maybe we should just not have this at all), maybe it should have been ISO format `2024-11-20T00:00:00Z` so that it's standard formatted and easily parseable

Reply:
Oh I think your are suggesting iso
https://github.com/expressjs/morgan/blob/efd6bff1adae9418f12a8827614336c72e2c4300/index.js#L273